fdbserver/BlobManifest.actor.cpp

```diff
@@ -117,8 +167,26 @@ class BlobManifestDumper : public ReferenceCounted<BlobManifestDumper> {
 		return wr.toValue();
 	}
 
+	// Remove old manifest file
+	ACTOR static Future<Void> cleanup(Reference<BlobManifestDumper> self) {
+		state Reference<BackupContainerFileSystem> writer;
+		state std::string fullPath;
+		std::tie(writer, fullPath) = self->blobConn_->createForWrite(MANIFEST_FOLDER);
+		std::vector<BlobManifestFile> files = wait(BlobManifestFile::list(writer));
```

Comment:
This is fine for now, but will need to be changed for tenants where each tenant has a separate storage location, since the manifest currently isn't specific to each tenant.

Reply:
yes. it's on the plan. each tenant will have a different manifest folder, and we'll have a top level manifest file to point to the location of each tenant's manifest.